CppSamples/UtilityNetwork/TraceUtilityNetwork/TraceUtilityNetwork.cpp

```diff
@@ -186,10 +190,11 @@ void TraceUtilityNetwork::connectSignals()
     constexpr double tolerance = 10.0;
     constexpr bool returnPopups = false;
     m_clickPoint = m_mapView->screenToLocation(mouseEvent.position().x(), mouseEvent.position().y());
-    m_mapView->identifyLayersAsync(mouseEvent.position(), tolerance, returnPopups).then(this, [this](const QList<IdentifyLayerResult*>& results)
+    auto f = m_mapView->identifyLayersAsync(mouseEvent.position(), tolerance, returnPopups).then(this, [this](const QList<IdentifyLayerResult*>& results)
```

Comment:
In some places we store `f` and pass it to `addTask`, and in others we just pass the future in place with the result of the function call (like on line 448). I'm wondering if readability would improve or suffer if we just do this one way instead of both, what do you think?

Reply:
My intent was to try to have this be transparent and not bloat or complicate the sample workflow(s). We can do it either way, so I will standardize it for consistency.